THIRD_PARTY_LICENSES.txt

```diff
@@ -2591,6 +2591,36 @@ SOFTWARE.
 
 ------
 
+** requests-aws4auth 1.0.1; version 1.0.1 --
```

Comment:
Did we need to add boto3 in here as well?

Reply:
Boto3 and botocore are both already added as they were test dependencies